config/javascript.js

```diff
@@ -42,22 +41,20 @@ gulp.task('js:vendor', function() {
  * Watch scripts for changes and move to the "static" folder
  */
 
-gulp.task('js:watch', function() {
-  gulp.watch('src/js/**/*.js', ['js:concat', 'js:vendor']);
+gulp.task('js:watch', async function() {
```

Comment:
I'm pretty sure we're good on the use of `async` anywhere we build these files. I looks like default support for `async`/`await` was added in node 8.x. Just something to think about as we're setting up new CI stuff. We'll need a minimum of of 8.x version to build.

As an alternative you can also pass a function to the call back here and eliminate the need for using async:

```js
gulp.task('js:watch', function(callback) {
  // all you task stuff here...

  callback();
}